cmd/lora-app-server/cmd/configfile.go

```diff
@@ -62,6 +62,13 @@ dsn="{{ .PostgreSQL.DSN }}"
 # App Server and / or applying migrations.
 automigrate={{ .PostgreSQL.Automigrate }}
 
+# Max open connections in the pool.
+# <= 0 means unlimited
+max_open={{ .PostgreSQL.MaxOpen }}
```

Comment:
One comment, would you mind renaming this to `max_open_connections` (and `MaxOpenConnections`) and `max_idle_connections` (and `MaxIdleConnections`)? That way the configuration variables are a bit more descriptive. After that I'll merge your changes.

Reply:
Fields renamed.

Reply:
Pull request for LoRaServer here: [loraserver pr 437](https://github.com/brocaar/loraserver/pull/437)